src/main/java/games/strategy/engine/ClientContext.java

```diff
@@ -50,6 +54,8 @@
   private final BattleCalcSettings battleCalcSettings;
   private final BattleOptionsSettings battleOptionsSettings;
 
+  private final DownloadCoordinator downloadCoordinator = new DownloadCoordinator();
```

Comment:
Seems fine here overall. I'd point out we do all the other initialization in the constructor. This class is 'all about' initialization (or at least is meant to be, the primary responsibility of it is to instantiate stuff in the right order and hold references to it).

So I'd comment that we may want to keep the init cohesive with-in the constructor - otherwise LGTM here.